pkg/costattribution/manager.go

```diff
@@ -176,7 +190,12 @@ func (m *Manager) ActiveSeriesTracker(userID string) *ActiveSeriesTracker {
 		return strings.Compare(a.Input, b.Input)
 	})
 
-	tracker = NewActiveSeriesTracker(userID, labels, maxCardinality, cooldownDuration, m.logger)
+	tracker, err := NewActiveSeriesTracker(userID, labels, maxCardinality, cooldownDuration, m.logger)
+	if err != nil {
+		m.invalidTrackers.WithLabelValues(userID, activeSeriesTracker).Inc()
+		return nil
+	}
+	m.invalidTrackers.WithLabelValues(userID, activeSeriesTracker).Set(0)
```

Comment:
This metric is a counter, we should not reset it to 0. If it grows, it's an issue, if it stays constant, it's ok.